webapp/providers/github.py

```diff
@@ -27,6 +30,13 @@ def verify_signature(headers, data, secret):
     return True
 
 
+def process_response(response):
+    if not response.ok:
+        raise Exception("Error from api: " + response.status_code)
```

Comment:
Is there a string value in the body of what the error is? Maybe that should be surfaced as well.

Reply:
I might be able to update this too have something more litteral. I don't want to make somethign too complex since here if the API misses I don't wnat ot have something to complex.

Reply:
I'll use talikser for error status. Here I am not really logging anything, so this will be lost.